### DISCUSSION TITLE
Non-Snap versions for Linux?

### DISCUSSION THREAD

**Discussion by aadrian** [massCodeIO/massCode#350] [General]
Hi,

Can we please have non-snap versions for Linux (too) ?

The GitHub Actions/ElectronBuilder that builds this Electron app are **perfectly able** to build Electron apps as non-snap too: https://www.electron.build/configuration/linux

> Target package type: list of:
```
AppImage, flatpak, snap, deb, rpm, freebsd, pacman ....
```

Thank you.

**Comment by ghost** [comment]
You can mount the .snap (sudo mount -t squashfs -o ro /pat/to/.snap /where/it's/mounted) then pull the files out. Does seem to run fine outside of snap. Probally could repackage it for flatpak from there.

**Reply by aadrian** [reply]
@PoorMeAnotherDrink Sorry, but that's not the point here. Since this project is built automatically by the CI/CD, that can be trivially extended with further artifact types as mentioned above.
